### DISCUSSION TITLE
lyric always faild

### DISCUSSION THREAD

**Discussion by OmiChrome** [clangen/musikcube#584] [Q&A] [answered]
:( whenever im using it lyric fails even i have lyric in folder in current directory of track
![image](https://user-images.githubusercontent.com/108952834/221962355-0a8ee566-b80e-48cb-abee-7ce33c5b2917.png)
only this is showing   , tried so many times 

OS: win 11 
installer : scoop 
terminal : window - terminal 
powershell core 7

**Comment by pietro1704** [comment, marked as answer]
I've figured it out. You need to register to [Audd.io - https://audd.io/](https://audd.io/) to get an API key, and paste this key in your advanced settings config:
<img width="853" alt="Captura de Tela 2023-04-05 às 15 12 49" src="https://user-images.githubusercontent.com/45739756/230168214-fdd20efd-03c9-40fb-a827-5168babc7ea8.png">
<img width="780" alt="Captura de Tela 2023-04-05 às 15 13 02" src="https://user-images.githubusercontent.com/45739756/230168272-678a695d-e797-4d80-aec2-c69cb36ecdac.png">


Unfortunately, not all songs have lyrics found by audd.io, and the free api key expires. But it's nice to see it working: 
<img width="534" alt="Captura de Tela 2023-04-05 às 15 11 25" src="https://user-images.githubusercontent.com/45739756/230167721-b6d85aa7-a308-47c5-a895-f4a7b7d2b3f2.png">

**Reply by OmiChrome** [reply]
it doesn't work for me coz i dont have subscribtion even trial

so please switch to free-lyric provider
